locales/en/translation.json

```diff
@@ -311,8 +311,10 @@
   "request_error":"Request Error",
   "invalid_transaction_hash": "Invalid Transaction Hash",
   "seed_character_set": "Only uppercase latin letters and the number 9 are allowed.",
-  "double_spend_warning": "Double Spend Warning",
-  "double_spend_attempt": "You are attempting to spend an already used input. You should wait for your transaction to be confirmed first before attempting to make a conflicting transaction. Do you want to continue anyway?",
+  "key_reuse_error": "Error: Private Key Reuse Detected!",
+  "key_reuse_attempt": "You are attempting to sign a transaction with input that have an already been used. IOTA uses Winternitz one-time signature (W-OTS) scheme, due to it's `one-time` nature, the security of funds in an address decreases rapidly if you re-sign new transactions using the same key.Please wait for previous transaction to confirm, before sending another transaction.",
```

Comment:
missing whitespace after `key.`
also, we should request them to reattach `... to confirm, e.g. by reattaching, before sending ...`

Reply:
done.